docs/source/pythonapi/mgxs.rst

```diff
@@ -4,5 +4,38 @@
 Multi-Group Cross Sections
 ==========================
 
-.. automodule:: openmc.mgxs.mgxs
-    :members: MGXS
+.. autoclass:: openmc.mgxs.mgxs.MGXS
```

Comment:
Along the same lines, do you know if it is (easily) possible to list a "table of contents" at the top of the page with links to each of the classes below?